src/main/kotlin/anttracker/InputHelpers.kt

```diff
@@ -0,0 +1,41 @@
+package anttracker
```

Comment:
I think this is a great file, and I don't mind if you use these. But I think it is unfair if we expect everyone on the team to use/follow these - we don't have too much time.

Reply:
This PR reflects what I would like to use. I added these functions in a file because I think they are useful. If nobody thinks so, I can move these functions to the `Issue.kt` file for my own use.

Reply:
Sounds good